### PR TITLE
Add delegate statistics page - Closes #751

### LINKED ISSUE #751: Implement delegate detail page - MIke *hard

### PR DESCRIPTION
**Pull request by michaeltomasik**
### What was the problem?
No implementation of Delegate statistics page
### How did I fix it?
Implemented it according to design https://taikonauten.invisionapp.com/d/main/#/console/12665756/291763570/preview
https://taikonauten.invisionapp.com/d/main/#/console/12665756/291764731/inspect
https://taikonauten.invisionapp.com/d/main/#/console/12665756/291764730/inspect
https://taikonauten.invisionapp.com/d/main/#/console/12665756/293437729/inspect
### How to test it?
Search in explorer for delegate ex. `5201600508578320196L` and compare Delegate statistics to design
### Review checklist
- The PR solves #751
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ module.exports = function (config) {
             'src/components/onboarding/index.js',
             'src/components/passphrase/createSecond/index.js',
             'src/components/passphraseCreation/index.js',
+            'src/components/accountTransactions/index.js',
```
Comment (ginacontrino) [inline karma.conf.js]: Why no tests for this one?

File: src/components/transactions/delegateStatistics.css [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    &.dirty {
+      border-bottom-color: var(--color-grayscale-medium);
+
+      & ~ .clean {
+        visibility: visible;
+      }
+    }
+  }
+
+  & .clean,
```
Comment (ginacontrino) [inline src/components/transactions/delegateStatistics.css]: Why are the styles implemented, but no actual icon for this? When I search, I currently can't clear my search

File: src/components/transactions/delegateStatistics.test.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      }),
+    },
+  };
+
+  it('should render DelegateStatistics', () => {
+    const wrapper = mountWithContext(<DelegateStatistics />, {});
+    expect(wrapper.find('DelegateStatistics').length).to.have.equal(1);
+  });
+
+  it('should not display showAll button', () => {
```
Comment (ginacontrino) [inline src/components/transactions/delegateStatistics.test.js]: can you please be a bit more specific about what this is testing? it is confusing to read  that it should not display the button, when initially we expect it to be shown (which you are testing). e.g. 
`should not show the "show more" button, if all votes are already shown`

File: src/components/transactions/delegateStatistics.test.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    const wrapper = mountWithContext(<DelegateStatistics />, { storeState });
+
+    expect(wrapper.find('.showAll').length).to.have.equal(1);
+    wrapper.find('.showAll').simulate('click');
+    expect(wrapper.find('.showAll').length).to.have.equal(0);
+  });
+
+  it('should not display showMore button', () => {
```
Comment (ginacontrino) [inline src/components/transactions/delegateStatistics.test.js]: Similar here, please be more specific under which condition the button should be shown & under which conditions it shouldn't be shown + please write the test for that as you did in the test above

**Comment by slaweet** [comment]
screen shot 2018-05-09 at 13 54 16
The white box should have padding-bottom as big as left/right .

File: src/components/accountTransactions/index.js [hunk 5]
```diff
@@ -33,21 +51,27 @@ class accountTransactions extends React.Component {
...
   publicKey: state.account.publicKey,
   activePeer: state.peers.data,
   notLoading: state.loading.length === 0,
+  peers: state.peers,
+  delegate: state.account && state.account.delegate,
 });
 
 const mapDispatchToProps = dispatch => ({
+  accountVotersFetched: data => dispatch(accountVotersFetched(data)),
+  accountVotesFetched: data => dispatch(accountVotesFetched(data)),
```
Comment (slaweet) [inline src/components/accountTransactions/index.js]: https://jenkins.lisk.io/job/lisk-hub/job/PR-765/20/cobertura/src_components_accountTransactions/index_js/
These two lines not being covered tell me that integration tests are missing. If too much work, integration tests can be a separate PR.

File: test/e2e/explorer.feature [hunk 6]
```diff
@@ -52,3 +52,27 @@ Feature: Explorer page
...
+    And I wait 1 seconds
+    When I click "explorer" menu
+    When I fill in "4401082358022424760L" to "search input" field
+    And I click "input search button"
+    And I wait 3 seconds
+    And I click "delegate statistics"
+    And I wait 1 seconds
+    Then I should see 35 instances of "votesFilterQuery row"
+    When I click "show votes"
+    Then I should see 101 instances of "votesFilterQuery row"
```
Comment (slaweet) [inline test/e2e/explorer.feature]: IMO there should be no new scenario. It should be part of the previous scenarios, to avoid most of the setup. You can even use `4401082358022424760L` instead of `16313739661670634666L` from the other scenarios.

**Comment by michaeltomasik** [comment]
Added follow up for tests 
https://github.com/LiskHQ/lisk-hub/issues/801
